labs/llm/context.py

```
@@ -1,15 +1,40 @@
+import logging
+import mimetypes
 from typing import List
 
-from file_handling import get_file_contents
+from file_handler import get_file_content
 
-CONTENT_TEMPLATE = "File: {file}, Content: {content}"
+logger = logging.getLogger(__name__)
 
+MIMETYPE_MD_NAME = {
+    "text/plain": "plaintext",
+    "application/json": "json",
+    "text/html": "html",
+    "text/x-python": "python",
+}
+CONTENT_TEMPLATE = "The following is the Python code in `{file}`:\n\n````{mimetype}\n{content}\n```"
 
-def get_context(file_paths: List[str], prompt: str):
+
+def get_file_mimetype(file_path: str) -> str:
+    if not file_path:
+        return ""
```

if you're trying to get the mimetype of a file without a path, shouldn't this raise an exception instead of returning ""?